apps/sms/views/conversation/js/compose.js

```diff
@@ -733,7 +733,14 @@ var Compose = (function() {
           dom.message.removeChild(currentAttachmentDOM);
           disposeAttachmentNode(currentAttachmentDOM);
           onContentChanged(newAttachment);
-        }, this._onAttachmentRequestError);
+        }, (err) => {
+          var errId = err instanceof DOMError ? err.name : err.message;
```

Comment:
__question:__ I'm wondering if we should hide replace menu in case of error at all? I see previously if you e.g. choose video and then will try to replace it with too-big one - you'll see error dialog and menu will stay open. It's edgy case, but I'm not sure what is better from UX perspective, could you please confirm with Morpheus if you have him handy? :) But anyway, I won't block on this.